### ISSUE TITLE
Edits to documents should have a color showing who made the edit

### ISSUE THREAD

**Issue by dessalines** [hackmdio/hackmd#588]
When collaborating on a document, its impossible to see who made the addition / edit. Other systems like piratepad do this by showing a color next to the editor, and providing a way for those without a sign in to name themselves.

**Comment by SISheogorath** [comment]
Sounds like a duplicate of #268 and #503 
Can you confirm this?

**Comment by dessalines** [comment]
Yes it looks like they are, but I've never seen the color-coding when testing out anonymous editors.

**Comment by SISheogorath** [comment]
in case of anonymous coding there is no color coding. Since we don't store the authorship in the database for guests

**Comment by dessalines** [comment]
hrm.... that seems like a bad idea, especially if you want to do consistent naming. Just make them an unverified user and set a cookie on their browser.

**Comment by SISheogorath** [comment]
There is a space for improvements. No doubt.

That's why ideas and pull requests to tackle these issues are welcome :+1: 

I would say that #268 is a good point to start, since it's not too difficult to implement and it's a good way to understand how guests are currently handled in HackMD.

Providing detailed information who edited what and when needs way more work in backend and is nothing we will be able to implement over night.
